scripts/ccf/member/add.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+REPO_ROOT="$(realpath "$(dirname "$(realpath "${BASH_SOURCE[0]}")")/../../..")"
+
+ccf-member-add() {
+    set -e
```

Comment:
should we delete create_member_cert_akv.sh?

Reply:
Are there any existing workflows that anyone has which relies on this? If not then I think we could, if we're not sure I would lean to keeping it to avoid breaking anyone?

Reply:
We can probably remove it eventually either way

Reply:
no this was just for setting the first key in akv. If we delete it we need to change docs to point to the new scripts.

Reply:
Happy to remove then if nothing uses it
https://github.com/microsoft/azure-privacy-sandbox-kms/pull/240/commits/eb4a4a5a6ada3b7b3525293e0f01ffab4c075798

Reply:
please update docs to points to the new script

Reply:
I did its in the commit mentioned above

Reply:
docs/Create_new_akv_member_key.md still references create_member_cert_akv

Reply:
That file is deleted in the change I mentioned above (specifically I renamed it to `Adding_members.md` since it applies to both AKV and non AKV cases, and git sees it as deleting the old file and replacing with the new one)